test/integration/auction.test.js

```diff
@@ -14,7 +14,7 @@ const { createWallet } = require("../../models/wallets");
 const userData = require("../fixtures/user/user")();
 const { auctionData, auctionKeys, auctionWithIdKeys } = require("../fixtures/auctions/auctions");
 const { initial_price: initialPrice, item_type: itemType, end_time: endTime, quantity } = auctionData;
-const currenciesData = require("../fixtures/currencies/currencies");
+const currenciesData = require("../fixtures/currencies/currencies")()[0];
```

Comment:
This is a little complex to understand. You can define the `[0]` data to another variable on another line?

something like `const currencyData = currenciesData()[0]`

Also I think this could have been done without making `currenciesData` a function

Reply:
1) Will do that. :+1: 

2) Mentioned above